pkts-sip/src/test/java/io/pkts/packet/sip/impl/SipParserTest.java

```diff
@@ -133,6 +133,7 @@ public void testConsumeUserInfoHostNoUserPart() throws Exception {
      */
     @Test
     public void testConsumeUserInfoHostInvalidHostChars() throws Exception {
+        assertConsumeUserInfoHostThrowsParseException("127.0.0:5060");
```

Comment:
please add unit tests for actually verifying that you do calculate the correct index...